src/openforms/forms/api/serializers/form_definition.py

```diff
@@ -114,8 +114,7 @@ class Meta:
                 "view_name": "api:formdefinition-detail",
                 "lookup_field": "uuid",
             },
-            # TODO: enable this in v3, deprecate writing this field
-            # "name": {"read_only": True},  # writing is done via the `translations` field
+            "name": {"read_only": True},  # writing is done via the `translations` field
```

Comment:
@sergei-maertens do I understand it correctly that this field is currently no longer used for writing? I was wonder whether the import code needs to add the name in translations if it doesn't exist. Or can we assume that imports have these translations already?

Reply:
I added a changelog entry for it